test/integration/auth/rbac_test.go

```diff
@@ -680,3 +683,66 @@ func TestBootstrapping(t *testing.T) {
 	}
 	t.Errorf("error bootstrapping roles: %s", string(healthBytes))
 }
+
+func TestDiscoveryUpgradeBootstrapping(t *testing.T) {
+	var tearDownFn kubeapiservertesting.TearDownFunc
+	defer func() {
+		if tearDownFn != nil {
+			tearDownFn()
+		}
+	}()
+
+	// Pre-populate etcd with the old default RBAC discovery bindings
+	etcdConfig := framework.SharedEtcd()
+	server := kubeapiservertesting.StartTestServerOrDie(t, nil, nil, etcdConfig)
+	tearDownFn = server.TearDownFn
```

Comment:
This variable was only needed in my test because I was restarting the apiserver and bringing it up again, but it doesn't look like your test does that. Do you plan to add that in the future? If not, you can just say `defer server.TearDownFn()`  and get rid of the variable

Reply:
ditto